sync/class.jetpack-sync-module-woocommerce.php

```diff
@@ -113,4 +118,115 @@ public function estimate_full_sync_actions( $config ) {
 	private function get_where_sql( $config ) {
 		return '1=1';
 	}
+
+	public function options_whitelist( $list ) {
```

Comment:
Sometimes phpdocs and comments are necessary, but I'm a big fan of "self-documenting" code whenever possible.